src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -90,4 +99,37 @@ private String setDisplayCaseReferenceFlag(String caseRef, Classification classi
             .map(CcdCollectionElement::new)
             .collect(toList());
     }
+
+    private String extractSurnameFromOcrData(Envelope envelope) {
+        if (CollectionUtils.isEmpty(envelope.ocrData)) {
+            return null;
+        }
+
+        String surnameOcrFieldName = serviceConfigProvider.getConfig(envelope.container)
+            .getSurnameOcrFieldName(envelope.formType);
+
+        List<String> surnameList = envelope.ocrData.stream().filter(ocrData -> ocrData.name.equals(surnameOcrFieldName))
```

Comment:
let's put `.stream()` and `.filter()` to new lines so it nicely matches the `.map` the line below